scss/_patterns_segmented-control.scss

```diff
@@ -1,9 +1,12 @@
-@mixin vf-p-tab-buttons {
+@mixin vf-p-segmented-control {
+  .p-segmented-control,
   .p-tab-buttons {
```

Comment:
A comment around here saying that `p-tab-buttons` is deprecated and please use segmented button instead would be helpful (for us to later easier find the deprecated code when we want to remove it).

Reply:
Done!